test/components/mods/RandomHeldItemsMod.ts

```diff
@@ -0,0 +1,40 @@
+import { INewPokemon } from "./../../../src/components/constants/Pokemon";
+import { FullScreenPokemon } from "./../../../src/FullScreenPokemon";
+import { it } from "./../../main";
+import { stubBlankGame } from "./../../utils/fakes";
+
+const pokemonTitle: string[] = "CHARMANDER".split("");
+
+it("gives a Pokemon an item when generated probability is valid", (): void => {
+    // Arrange
+    const fsp: FullScreenPokemon = stubBlankGame();
+    fsp.modAttacher.enableMod("Random Held Items");
+    fsp.numberMaker.randomReal1 = (): number => .012;
```

Comment:
Make a helper method to create a new `fsp` with the mod and random function stubbed out. You can reduce the LoC in each test by 2.